src/test/java/uk/gov/hmcts/reform/ccd/document/am/service/impl/DocumentManagementServiceImplTest.java

```diff
@@ -454,45 +451,39 @@ private void verifyCaseDataServiceGetDocMetadata() {
     @Test
     @SuppressWarnings("unchecked")
     void patchDocumentMetadata_HappyPath() {
```

Comment:
We can have more test cases to check various conditions for new code : generateHashToken()

Reply:
yes, those can be added.